controller/tests/test_plan_controller.py

```diff
@@ -0,0 +1,147 @@
+import unittest
+from datetime import date
+
+from models.camp import Camp
+from models.plan import Plan
+from controller.plan_controller import close_plan
+from controller.plan_controller import find_plan
+from controller.plan_controller import create_plan
+from controller.plan_controller import list_plans
+from controller.plan_controller import create_camps
+from controller.plan_controller import controller_error
+
+
+class PlanControllerTest(unittest.TestCase):
+    def setUp(self):
+        Plan.delete_all()
+
+    def test_plan_status(self):
+        """
+        Test status flag of plan. Should be false is plan is open.
+        """
+        plan = Plan(name='My Plan',
+                    emergency_type=Plan.EmergencyType.EARTHQUAKE,
+                    description='Test emergency plan',
+                    geographical_area='',
+                    camps=[Camp(name='TestCamp')])
+        self.assertEqual(False, plan.close_plan_status)
+        self.setUp()
+
+    def test_close_plan_controller(self):
+        """
+        Test status flag of plan. Should be true when plan is closed.
+        """
+        plan = Plan(name='My Plan',
+                    emergency_type=Plan.EmergencyType.EARTHQUAKE,
+                    description='Test emergency plan',
+                    geographical_area='',
+                    camps=[Camp(name='TestCamp')])
+        close_plan(plan)
+        self.assertEqual(True, plan.close_plan_status)
+        self.setUp()
+
+    def test_no_close_date(self):
```

Comment:
Same here, it should be a class test. 
Also I'd name it `test_initiallised_plan_no_close_date`